Examples/Swift/AnnotationMovementExample.swift

```diff
@@ -0,0 +1,170 @@
+//  Copyright © 2016 Mapbox. All rights reserved.
+//
+
+import Mapbox
+
+@objc(AnnotationMovementExample_Swift)
+
+// Example view controller
+class AnnotationMovementExample_Swift: UIViewController, MGLMapViewDelegate {
+    var mapView: MGLMapView!
+    var path: [CLLocationCoordinate2D]!
+    var point: MGLPointAnnotation?
+    var pointCount = 1
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.FlexibleWidth, .FlexibleHeight]
+        mapView.styleURL = MGLStyle.outdoorsStyleURLWithVersion(9)
+        mapView.delegate = self
+        view.addSubview(mapView)
+
+        drawPolyline()
+    }
+
+    // MARK: - MGLMapViewDelegate methods
+
+    // This delegate method is where you tell the map to load a view for a specific annotation. To load a static MGLAnnotationImage, you would use `-mapView:imageForAnnotation:`.
+    func mapView(mapView: MGLMapView, viewForAnnotation annotation: MGLAnnotation) -> MGLAnnotationView? {
+        // This example is only concerned with point annotations.
+        guard annotation is MGLPointAnnotation else {
+            return nil
+        }
+
+        // For better performance, always try to reuse existing annotations. To use multiple different annotation views, change the reuse identifier for each.
+        if let annotationView = mapView.dequeueReusableAnnotationViewWithIdentifier("bicycle") {
+            return annotationView
+        } else {
+            return MoveableAnnotationView(reuseIdentifier: "bicycle", size: 10)
+        }
+    }
+
+    func moveBicycleAlongPath() {
+        if path == nil {
+            return
+        }
+
+        if point == nil {
+            point = MGLPointAnnotation()
+            point!.coordinate = path.first!
+            mapView.addAnnotation(point!)
+
+            let _ = NSTimer.scheduledTimerWithTimeInterval(0.25, target: self, selector: #selector(self.moveBicycleAlongPath), userInfo: nil, repeats: true)
+
+            return
+        }
+
+        if pointCount == path.count {
+            pointCount = 0
+        }
+
+        point?.coordinate = path[pointCount]
+        pointCount += 1
+
+        mapView.setCenterCoordinate((point?.coordinate)!, animated: true)
+    }
+
+    func drawPolyline() {
+        // Parsing GeoJSON can be CPU intensive, do it on a background thread.
+        dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), {
+            // Get the path for london.geojson in the app’s bundle.
+            let jsonPath = NSBundle.mainBundle().pathForResource("london", ofType: "geojson")
+            let jsonData = NSData(contentsOfFile: jsonPath!)
+
+            do {
+                // Load and serialize the GeoJSON into a dictionary filled with properly-typed objects.
+                if let jsonDict = try NSJSONSerialization.JSONObjectWithData(jsonData!, options: []) as? NSDictionary {
```

Comment:
This code needs to be eviscerated along the lines of #78, now that GeoJSON deserialization is built into the SDK.